src/models/training.py

```diff
@@ -0,0 +1,56 @@
+import sys
+sys.path.append('..')
+sys.path.append('.')
+
+import logging
+
+from corpus import HackernewsCorpus
+from gensim import utils, models
+from src.data.load_data import get_hackernews_files, load_hackernews_dataframe
+
+logging.basicConfig(format="%(asctime)s : %(levelname)s : %(message)s", level=logging.INFO)
+
+
+def training():
+    # Load one HackerNews DataFrame
+    files = get_hackernews_files()
+    df = load_hackernews_dataframe(files[-1])
+
+    # Focus on the stories
+    comments_df = df[df.type == "comment"]
+    comments_df = comments_df.dropna(subset=['text'])
+    comments = comments_df['text']
+
+    # Set up the streamed corpus
+    corpus = HackernewsCorpus(comments)
+
+    # Train 10 LDA topics using MALLET
+    mallet_path = '/home/madness/Programs/mallet-2.0.6/bin/mallet'
```

Comment:
Can you make this hardcoded path relative? I'm guessing not everyone is named Madness ;)

Reply:
So rename yourself :)